test/utils.js

```diff
@@ -56,27 +60,46 @@ function chunkBigInt(n, mod=BigInt(2**51)){
 	if (!n) return [0];
 	let arr = [];
 	while (n) {
-		arr.push(BigInt(n % mod));
+		arr.push(BigInt(modulus(n,mod)));
 		n /= mod;
 	}
 	return arr;
 }
-// This function will perform point addition on elliptic curve 25519 to check point addition circom
+
 let p = BigInt(2**255) - BigInt(19);
 let d = 37095705934669439343138083508754565189542113879843219016388785533085940283555n;
-
+// This function will perform point addition on elliptic curve 25519 to check point addition circom
 function point_add(P,Q){
-    let A =  (P[1]-P[0])*(Q[1]-Q[0])%p;
-    let B =  (P[1]+P[0])*(Q[1]+Q[0])%p; 
-    let C =  BigInt(2)*P[3]*Q[3]*d % p;
-    let D =  BigInt(2)*P[2]*Q[2] % p;
+  let A =  modulus((P[1]-P[0])*(Q[1]-Q[0]),p);
```

Comment:
Please use consistent formatting, sometimes you use 2 spaces, sometimes 4 spaces. Pick one and stick with it